### PR TITLE
Added a description of EvidenceStatementSet and why it is empty

### LINKED ISSUE #61: EvidenceStatementSet definition
Ned pointed out that the "Empty for now" comment is a bit misleading. It should probably be "None defined in this document" or similar.

Should that be a SET?

We should have some text explaining why this is empty here and how we envision new things (TGC / proprietary) things getting added.

```
   EvidenceStatementSet EVIDENCE-STATEMENT ::= {
      ... -- Empty for now --
   }

   EvidenceStatement ::= SEQUENCE {
      type   EVIDENCE-STATEMENT.&id({EvidenceStatementSet}),
      stmt   EVIDENCE-STATEMENT.&Type({EvidenceStatementSet}{@type})
   }
```

### PR DESCRIPTION
**Pull request by ounsworth**
Closes #61

### CODE REVIEW

**Comment by ounsworth** [comment]
This is addressing Ned Smith's comment, so he should review the change.

File: draft-ietf-lamps-csr-attestation.md [hunk 1]
```diff
@@ -402,10 +402,21 @@ certificate chain.
...
 }
+~~~
+
+This is a mapping and ASN.1 Types for Evidence Statements to the OIDs
+that identify them. These mappings are are used to construct
+or parse EvidenceStatements. These would typically be Evidence Statement
+formats defined in other IETF standards, defined by other standards bodies,
+or vendor proprietary formats along with the OIDs that identify them.
 
+This list is left empty in this document, however implementers will wish
```
Comment (hannestschofenig) [inline draft-ietf-lamps-csr-attestation.md:416]:
```suggestion
This list is left empty in this document. However, implementers should
```

File: draft-ietf-lamps-csr-attestation.md [hunk 2]
```diff
@@ -402,10 +402,21 @@ certificate chain.
...
+to populate it with the formats that they wish to support.
```
Comment (hannestschofenig) [inline draft-ietf-lamps-csr-attestation.md:417]:
```suggestion
populate it with the formats that they wish to support.
```

File: draft-ietf-lamps-csr-attestation.md [hunk 3]
```diff
@@ -402,10 +402,21 @@ certificate chain.
 ~~~
 EVIDENCE-STATEMENT ::= TYPE-IDENTIFIER
 
-EvidenceStatementSet EVIDENCE-STATEMENT ::= {
+EvidenceStatementSet EVIDENCE-STATEMENT ::= SET {
    ... -- Empty for now --
```
Comment (hannestschofenig) [inline draft-ietf-lamps-csr-attestation.md:406]:
```suggestion
   ... -- None defined in this document --
```

File: CSR-ATTESTATION-2023.asn [hunk 4]
```diff
@@ -58,7 +58,7 @@ TypedFlatCert ::= SEQUENCE {
 
 EVIDENCE-STATEMENT ::= TYPE-IDENTIFIER
 
-EvidenceStatementSet EVIDENCE-STATEMENT ::= {
+EvidenceStatementSet EVIDENCE-STATEMENT ::= SET {
    ... -- Empty for now --
```
Comment (hannestschofenig) [inline CSR-ATTESTATION-2023.asn:62]:
```suggestion
   ... -- None defined in this document --
```

**Pull request merged**